plex_generate_previews.py

```diff
@@ -8,23 +9,84 @@
 import glob
 import os
 import struct
-import gpustat
+
+if not os.path.isfile('/usr/bin/mediainfo'):
+    print('MediaInfo not found.  MediaInfo must be installed and available in PATH.')
+    sys.exit()	
+
+try:
+    from pymediainfo import MediaInfo
+except ImportError:
+    print('Dependencies Missing!  Please run "pip3 install pymediainfo".')
+    sys.exit()
```

Comment:
@jasonsansone can we add a 1 to all the sys.exit(). So sys.exit(1). The 1 will mean the program failed to execute.